apps/api/src/test-data/validateTestData.spec.ts

```diff
@@ -19,90 +22,172 @@ export type InMemorySnapshotOfDTOs = {
+                    });
 
-                const ResourceType = key as ResourceType;
-                it(`should have a corresponding collection name`, () => {
-                    const collectionName = getArangoCollectionIDFromResourceType(ResourceType);
+                    const ResourceType = key as ResourceType;
+                    it(`should have a corresponding collection name`, () => {
+                        const collectionName = getArangoCollectionIDFromResourceType(ResourceType);
 
-                    expect(isStringWithNonzeroLength(collectionName)).toBe(true);
-                });
+                        expect(isStringWithNonzeroLength(collectionName)).toBe(true);
+                    });
 
-                describe(`the DTOs`, () => {
-                    /**
-                     * TODO Also check that entitiy types that correspond to a
-                     * union of models have at least one example of each member
-                     * of the union. (See e.g. `Spatial Feature`)
-                     */
-                    it(`should have at least one test data DTO`, () => {
-                        const numberOfTestDataEntries = models.length;
+                    describe(`the DTOs`, () => {
+                        /**
+                         * TODO Also check that entitiy types that correspond to a
+                         * union of models have at least one example of each member
+                         * of the union. (See e.g. `Spatial Feature`)
+                         */
+                        it(`should have at least one test data DTO`, () => {
+                            const numberOfTestDataEntries = models.length;
 
-                        expect(numberOfTestDataEntries).toBeGreaterThan(0);
-                    });
-                    it(`should have no duplicate IDs`, () => {
-                        const allIds = models
-                            .map((model) => model.id)
-                            /**
-                             * We have a separate check for missing `id` properties
-                             */
-                            .filter((id) => !isNullOrUndefined(id));
+                            expect(numberOfTestDataEntries).toBeGreaterThan(0);
+                        });
+                        it(`should have no duplicate IDs`, () => {
+                            const allIds = models
+                                .map((model) => model.id)
+                                /**
+                                 * We have a separate check for missing `id` properties
+                                 */
+                                .filter((id) => !isNullOrUndefined(id));
 
-                        const numberOfIds = allIds.length;
+                            const numberOfIds = allIds.length;
 
-                        const numberOfUniqueIds = [...new Set(allIds)].length;
+                            const numberOfUniqueIds = [...new Set(allIds)].length;
```

Comment:
Here `Set()` ensures there are only unique ids in this array?

Reply:
Yup. See the pro-d channel in Slack for a breakdown.